moveit_setup_assistant/moveit_setup_framework/src/urdf_config.cpp

```diff
@@ -131,11 +144,6 @@ void URDFConfig::load()
     throw std::runtime_error("URDF/COLLADA file not found: " + urdf_path_.string());
   }
 
-  if (urdf_pkg_name_.empty())
```

Comment:
This exception should not be triggered if we actually don't provide a file path outside of a ROS package. That's why I moved it to `loadFromPackage()` above.